{
  "_from": "assertion-error@^1.0.1",
  "_id": "assertion-error@1.1.0",
  "_inBundle": false,
  "_integrity": "sha512-jgsaNduz+ndvGyFt3uSuWqvy4lCnIJiovtouQN5JZHOKCS2QuhEdbcQHFhVksz2N2U9hXJo8odG7ETyWlEeuDw==",
  "_location": "/assertion-error",
  "_phantomChildren": {},
  "_requested": {
    "type": "range",
    "registry": true,
    "raw": "assertion-error@^1.0.1",
    "name": "assertion-error",
    "escapedName": "assertion-error",
    "rawSpec": "^1.0.1",
    "saveSpec": null,
    "fetchSpec": "^1.0.1"
  },
  "_requiredBy": [
    "/chai"
  ],
  "_resolved": "https://registry.npmjs.org/assertion-error/-/assertion-error-1.1.0.tgz",
  "_shasum": "e60b6b0e8f301bd97e5375215bda406c85118c0b",
  "_spec": "assertion-error@^1.0.1",
  "_where": "/Users/Fossa/projects/go/src/github.com/fossas/fossa-cli/analyzers/nodejs/fixtures/with_npm_lockfile/node_modules/chai",
  "author": {
    "name": "Jake Luer",
    "email": "jake@qualiancy.com",
    "url": "http://qualiancy.com"
  },
  "bugs": {
    "url": "https://github.com/chaijs/assertion-error/issues"
  },
  "bundleDependencies": false,
  "dependencies": {},
  "deprecated": false,
  "description": "Error constructor for test and validation frameworks that implements standardized AssertionError specification.",
  "devDependencies": {
    "component": "*",
    "typescript": "^2.6.1"
  },
  "engines": {
    "node": "*"
  },
  "homepage": "https://github.com/chaijs/assertion-error#readme",
  "keywords": [
    "test",
    "assertion",
    "assertion-error"
  ],
  "license": "MIT",
  "main": "./index",
  "name": "assertion-error",
  "repository": {
    "type": "git",
    "url": "git+ssh://git@github.com/chaijs/assertion-error.git"
  },
  "scripts": {
    "test": "make test"
  },
  "types": "./index.d.ts",
  "version": "1.1.0"
}
